style.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  width: 375px;
+  margin: 0 auto;
+  background-color: brown;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+  height: 92px;
+  color: #fff;
+  background-color: #3c3a39;
+  margin: 0;
+  width: 100%;
+}
+
+nav {
+  height: 100%;
+  padding: 8px 14px;
+  margin: 0;
+}
+
+.app-bar {
+  display: flex;
+  justify-content: space-between;
+  line-height: 47px;
+}
+
+header nav .toolbar {
+  display: flex;
+  justify-content: space-between;
+  font-size: 18px;
+}
+
+.logo a {
+  font-size: 18px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.headline {
+  background-color: #1c1a19;
+  width: 100%;
+  padding-top: 120px;
+  padding-bottom: 70px;
+  margin: 0;
+  position: relative;
+}
+
+.headline h1 {
+  font-size: 56px;
+  line-height: 64px;
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
```

Comment:
- [ ] Kindly add the correct `<link>` tags to your HTML `head` to make that font work properly because now it doesn't work. This is the [link](https://fonts.google.com/specimen/Crete+Round?query=crete) to this font in google fonts.